packages/dds/tree/src/schema-stored/Stored and View Schema.md

```diff
@@ -35,104 +34,116 @@ Places we can store stored schema information:
 -   In some external repository: the repository defines an append only namespace of schema
     This repository is known about by shared-tree somehow (any of the other items in this list could contain the reference to the repository)
 
 ## Places that can refer to/apply stored schema:
 
--   Code can special case the root (ex: shared-tree, a subclass, or its configuration, can apply some rules, possibly recursively, to the root).
--   Declaratively for a specific type: nodes in the tree are typed, so similar to the above, the tree can be configured to apply specific schema based on type of the nodes.
-    This forces all nodes of the same type to have these same schema: its not contextual, but easily handles open polymorphism (ex: a child thats allowed to be anything, as long as its in schema for its type).
--   In other schema: for example, a schema can apply specific schema to its children.
+-   At the root (ex: shared-tree, a subclass, or its configuration), which can be special cased to apply some rules, possibly recursively, to the root.
+-   Declaratively on nodes of a specific type: Nodes in the tree are typed, so similar to the above, the tree can be configured to apply specific schema based on type of the nodes.
+
+    This forces all nodes of the same type to have the same schema: it's not contextual but easily handles open polymorphism (ex: a child that's allowed to be anything as long as it's in schema for its type).
+-   In other schema: For example, a schema can apply specific schema to its children.
+
     This allows for `contextual schema` (under one parent the same type might have different rules compared to under another parent).
 
-Note that its possible to refer to schema in a way that's unambiguous, but code handing the data might not always have the schema.
-For example document could refer to a schema by its hash, or name in a append only namespace.
+Note that it's possible to refer to schema in a way that's unambiguous, but code handling the data might not always have the schema.
+For example, document could refer to a schema by its hash, or name in an append only namespace.
 This can have interesting implications for updates to new schema (ex: one client adds data using a schema shipped as code that another client does not have).
 Some of the options do not have this issue (inline, central repository (assuming you are ok going down if it goes down and schema are not cached), and in stored data).
 
-# What options to support.
+# Options to Support
 
-Thats a lot of options, and I think there are good reasons for all of them, so long term, we likely want all of them to be options to some extent.
-But I think we can pick a good subset for the now, and leave open the option of extending it in the future.
+That's a lot of options and I think there are good reasons for all of them. Long term, we likely want all of them implemented to some extent but I think we can pick a good subset for now and allow extension in the future.
```

Comment:
This section should be adjusted to avoid the first person. This document is no longer a design proposal and using first person in it no longer makes sense (if it ever did).

```suggestion
That's a lot of options all of which have use-cases. Long term, at least basic support for all of them may make sense but a much smaller subset can be supported initially while supporting most use-cases pretty well.
```